src/app/tests/suites/certification/Test_TC_TSTAT_2_1.yaml

```diff
@@ -730,3 +730,97 @@ tests:
           value: 0
           constraints:
               type: enum8
+
+    - label: "Step 52: TH reads the PresetTypes attribute from the DUT"
+      PICS: TSTAT.S.F08 && TSTAT.S.A0048
```

Comment:
Bcoz of most of the below attributes are mandatory when the feature is present, I don't think you need both the PICS here. Just have one PICS TSTAT.S.F08, that way anyone implementing that feature must implement these attributes. You can update the same in the test plan too.

Reply:
addressed.